apps/system-e2e/src/tests/service-portal/smoke/company-delegation.spec.ts

```diff
@@ -39,7 +39,7 @@ test.describe('Service portal', () => {
       .textContent()
     expect(companyName).toBeTruthy()
     await firstCompany.click()
-    await page.waitForURL(homeUrl, {
+    await page.waitForURL(new RegExp(homeUrl), {
```

Comment:
A bit uncomfortable with converting a URL to a regex, but should work in this case anyway. A better way maybe to use simply `/minarsidur*`?